drivers/net/Kconfig

```diff
@@ -183,6 +183,11 @@ menuconfig NET_LOOPBACK
 
 if NET_LOOPBACK
 
+config NET_LOOPBACK_SIMULATE_PACKET_DROP
+	bool "Controlable packet drop"
+	help
+	  Enable interface to have a controlable packet drop rate
```

Comment:
It would be nice to emphasize that this is only used by testing and should not be enabled by end user.